packages/web/src/components/delete-confirmation/DeleteConfirmationModal.tsx

```diff
@@ -1,13 +1,28 @@
 import React, { memo } from 'react'
 
 import { Modal, Button, ButtonSize, ButtonType } from '@audius/stems'
-import PropTypes from 'prop-types'
 
 import styles from './DeleteConfirmationModal.module.css'
 
-const DeleteConfirmationModal = props => {
-  const header = `This ${props.entity} Will Disappear For Everyone`
-  const description = `Are you sure you want to delete this ${props.entity.toLowerCase()}?`
+type DeleteConfirmationModalProps = {
+  title: string
+  customHeader?: string
+  customDescription?: string
+  visible: boolean
+  entity: string
+  onDelete: () => void
+  onCancel: () => void
+}
+
+const DeleteConfirmationModal = (props: DeleteConfirmationModalProps) => {
+  const header =
+    props.customHeader == null
```

Comment:
I think we should be checking for undefined since the prop type is `string | undefined`

Reply:
Yeah I use `== null` (not to be confused with `=== null` to catch both null and undefined in cases where they would either `null` or `undefined` would mean the same thing (which is most of the time).